wagn/lib/wagn/generators/wagn/wagn_generator.rb

```diff
@@ -28,10 +28,10 @@ class WagnGenerator < Rails::Generators::AppBase
   
 ## should probably eventually use rails-like AppBuilder approach, but this is a first step.  
   def dev_setup
-    @gem_path = options['gem-path']
+    @gem_path = ENV['WAGN_DEV_GEM_PATH'] || options['gem-path']
```

Comment:
I think this may have been introduced on master and lost in a merge.  Here now in any case so we can do this either way.  Note the parameter version is used (and not verified in generated Gemfile) in the migration test script.